### PR TITLE
patch(*): removed null as acceptable value for complete and testable TRL

### PR DESCRIPTION
**Pull request by Daniel-Searle**
## Removed null as acceptable value for complete and testable TRL

_One line description_

[CB2-XXXX](https://dvsa.atlassian.net/browse/CB2-XXXX)

## Changelog
- Removed null as acceptable value for required fields in complete and testable TRL

### CODE REVIEW

**Review by gjulien-bjss — COMMENTED** [review]
You need to edit the schemas under the `json-definitions` folder, hence the failing checks

File: json-definitions/v3/tech-record/get/trl/testable/index.json [hunk 1]
```diff
@@ -719,9 +719,6 @@
           "type": "integer",
```
Comment (naathanbrown) [inline json-definitions/v3/tech-record/get/trl/testable/index.json:719]: Need to remove the any of if there is only one thing

File: json-definitions/v3/tech-record/get/trl/complete/index.json [hunk 2]
```diff
@@ -478,40 +478,35 @@
     },
     "techRecord_couplingCenterToRearAxleMax": {
       "type": [
-        "integer",
-        "null"
+        "integer"
```
Comment (naathanbrown) [inline json-definitions/v3/tech-record/get/trl/complete/index.json]: need to remove these from arrays to be just "type": "integer" for example if only one thing

**Comment by naathanbrown** [comment]
Can you also add a line to the change log too please :)

**Review by gjulien-bjss — APPROVED** [review]

**Review by naathanbrown — APPROVED** [review]

**Pull request merged**